src/Test/PHPUnit/AbstractCacheConstraint.php

```diff
@@ -62,6 +62,10 @@ protected function matches($other)
      */
     protected function failureDescription($other)
     {
-        return (string) $other . ' ' . $this->toString();
+        return sprintf(
+            'response (with status code %s) %s',
+            $other->getStatusCode(),
+            $this->toString()
```

Comment:
i think this is the last open question. anything else we need to discuss before merging?